doc/doc.sh

```diff
@@ -20,8 +20,14 @@ fi
 
 case $1 in
   prepare)
-    echo "Clone documentation framework"
-    git clone --depth 10 --single-branch --branch master https://github.com/kuzzleio/documentation.git framework/
+    if [ -d "framework" ]
+    then
+      echo "Pulling latest framework version"
+      bash -c "cd framework && git reset --hard HEAD~ && git pull"
```

Comment:
```suggestion
      bash -c "cd framework && git reset --hard HEAD~ && git checkout master && git pull"
```
We have to switch back to master before pulling (see this failing build: https://app.netlify.com/sites/doc-sdk-javascript/deploys/5d40611fbfdf50000a98d5a7)

Reply:
Good call :+1:

Reply:
@Aschen > done